ssg/constants.py

```diff
@@ -267,7 +269,7 @@
 
 MULTI_PLATFORM_LIST = ["rhel", "fedora", "rhv", "debian", "ubuntu",
                        "wrlinux", "opensuse", "sle", "ol", "ocp", "rhcos",
-                       "example", "eks", "alinux"]
+                       "example", "eks", "alinux", "uos"]
```

Comment:
First, your content doesn't build because of this:
```
[12/18] [uosv20-content] generating ssg-uosv20-cpe-dictionary.xml, ssg-uosv20-cpe-oval.xml
FAILED: ssg-uosv20-cpe-dictionary.xml ssg-uosv20-cpe-oval.xml /home/jcerny/work/git/scap-security-guide/build/ssg-uosv20-cpe-dictionary.xml /home/jcerny/work/git/scap-security-guide/build/ssg-uosv20-cpe-oval.xml 
cd /home/jcerny/work/git/scap-security-guide/build/uosv20 && env PYTHONPATH=/home/jcerny/work/git/scap-security-guide /usr/bin/python3 /home/jcerny/work/git/scap-security-guide/build-scripts/cpe_generate.py --product-yaml /home/jcerny/work/git/scap-security-guide/products/uosv20/product.yml ssg /home/jcerny/work/git/scap-security-guide/build /home/jcerny/work/git/scap-security-guide/build/uosv20/shorthand.xml /home/jcerny/work/git/scap-security-guide/build/uosv20/oval-unlinked.xml && /usr/bin/xmllint --nsclean --format --output /home/jcerny/work/git/scap-security-guide/build/ssg-uosv20-cpe-dictionary.xml /home/jcerny/work/git/scap-security-guide/build/ssg-uosv20-cpe-dictionary.xml && /usr/bin/xmllint --nsclean --format --output /home/jcerny/work/git/scap-security-guide/build/ssg-uosv20-cpe-oval.xml /home/jcerny/work/git/scap-security-guide/build/ssg-uosv20-cpe-oval.xml

	Error: Can't locate "proc_sys_kernel_osrelease_arch_aarch64" definition in "/home/jcerny/work/git/scap-security-guide/build/uosv20/oval-unlinked.xml".             
	Ensure <platform> element is configured properly for "proc_sys_kernel_osrelease_arch_aarch64".              
	Exiting..
[15/18] [uosv20-playbooks] generating Ansible Playbooks for all profiles in ssg-uosv20-xccdf.xml
ninja: build stopped: subcommand failed.
```

Second, if your product name is `uosv20` then you need to put `uosv` here, not `uos`. Check the `parse_name` function in `/ssg/utils.py` to see how this works. It would probably make sense to rename your product to `uos20` instead.

Reply:
Ｔhank you very much for your help. I discussed with our team according to your suggestions and decided to change our name．